snuba/datasets/processors/generic_metrics_processor.py

```diff
@@ -141,7 +141,13 @@ def process_message(
                 GRANULARITY_ONE_DAY,
             ],
         }
-        return InsertBatch([processed], None)
+        return InsertBatch(
+            [processed],
+            None,
+            sentry_received_timestamp=datetime.utcfromtimestamp(
+                message["sentry_received_timestamp"]
```

Comment:
I would suggest to make it optional for now to minimize risk

Reply:
ok sure, can do